README.md

```diff
@@ -153,11 +153,11 @@ Apart from implementing the abstract methods, you should make sure that the enti
 Regarding testing, you should have two instances of the game running at the same time and extensively test entering/leaving the scene of the entity at all possible states that the FSM can have.
  
 If you have any more questions or remarks, do not hesitate to post them in the [discord server](https://discord.gg/KbgxvDyzHP)'s #entity-system-development.
-If you want to start working on the implementation of an entity, either make a draft pull request or an issue, so we can keep track of who is doing what and do not accidentally overlap.
+If you want to start working on the implementation of an entity, either make a draft pull request, or an issue. That way we can keep track of who is currently working on which entity to avoid accidentally overlapping.
 When you finish the implementation and have tested it thoroughly, you can make a pull request.
 
 Current progress on the entity system can be seen on the [project page](https://github.com/Extremelyd1/HKMP/projects/1).
-If you decide to start implementing an entity you can either make a draft pull request or open an issue and you'll be added to the project.
+If you decide to start implementing an entity you can either, make a draft pull request, or open an issue and you'll be added to the project.
```

Comment:
I feel like there shouldn't be a comma behind "either" in this sentence. It stops the flow of reading the sentence awkwardly. Feel free to provide justification if this is indeed the correct use of commas in this case however.

Reply:
I totally agree, with your point.